packages/doc/docs/extensions/vulcan-sql-core.mdx

```diff
@@ -0,0 +1,16 @@
+# Extensions of the @vulcan-sql/core package
+
+The @vulcan-sql/core provide 10 extension components:
```

Comment:
```suggestion
The @vulcan-sql/core provides 10 extension components:
```

Reply:
Thanks for the suggestion, I have fixed the word